src/UserSettingsPage.cpp

```diff
@@ -286,113 +241,89 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
         int themeIndex = themeCombo_->findText(themeStr);
         themeCombo_->setCurrentIndex(themeIndex);
 
-        themeOptionLayout_->addWidget(themeLabel_);
-        themeOptionLayout_->addWidget(themeCombo_, 0, Qt::AlignRight);
-
-        auto encryptionLayout_ = new QVBoxLayout;
-        encryptionLayout_->setContentsMargins(0, OptionMargin, 0, OptionMargin);
-        encryptionLayout_->setAlignment(Qt::AlignVCenter);
+        auto encryptionLabel_ = new QLabel{tr("ENCRYPTION"), this};
+        encryptionLabel_->setSizePolicy(QSizePolicy::Ignored, QSizePolicy::Fixed);
+        encryptionLabel_->setFont(font);
 
         QFont monospaceFont;
         monospaceFont.setFamily("Monospace");
         monospaceFont.setStyleHint(QFont::Monospace);
         monospaceFont.setPointSizeF(monospaceFont.pointSizeF() * 0.9);
 
-        auto deviceIdLayout = new QHBoxLayout;
-        deviceIdLayout->setContentsMargins(0, OptionMargin, 0, OptionMargin);
-
-        auto deviceIdLabel = new QLabel(tr("Device ID"), this);
-        deviceIdLabel->setFont(font);
-        deviceIdLabel->setMargin(0);
         deviceIdValue_ = new QLabel{this};
         deviceIdValue_->setTextInteractionFlags(Qt::TextSelectableByMouse);
         deviceIdValue_->setFont(monospaceFont);
-        deviceIdLayout->addWidget(deviceIdLabel, 1);
-        deviceIdLayout->addWidget(deviceIdValue_);
-
-        auto deviceFingerprintLayout = new QHBoxLayout;
-        deviceFingerprintLayout->setContentsMargins(0, OptionMargin, 0, OptionMargin);
 
-        auto deviceFingerprintLabel = new QLabel(tr("Device Fingerprint"), this);
-        deviceFingerprintLabel->setFont(font);
-        deviceFingerprintLabel->setMargin(0);
         deviceFingerprintValue_ = new QLabel{this};
         deviceFingerprintValue_->setTextInteractionFlags(Qt::TextSelectableByMouse);
         deviceFingerprintValue_->setFont(monospaceFont);
-        deviceFingerprintLayout->addWidget(deviceFingerprintLabel, 1);
-        deviceFingerprintLayout->addWidget(deviceFingerprintValue_);
 
-        auto sessionKeysLayout = new QHBoxLayout;
-        sessionKeysLayout->setContentsMargins(0, OptionMargin, 0, OptionMargin);
-        auto sessionKeysLabel = new QLabel(tr("Session Keys"), this);
+        std::string fingerprintPlaceHolder_(44, 'x');
+        deviceFingerprintValue_->setText(utils::humanReadableFingerprint(fingerprintPlaceHolder_));
```

Comment:
```suggestion
        deviceFingerprintValue_->setText(QStringLiteral(44, 'X'));
```

Reply:
did something like this, but still using humanreadableFingerprint so it formats it the same way in quartets

Reply:
Ah, yeah, I messed that up, sorry :D